cosmic-usage-ui/src/main/resources/public/app/main.css

```diff
@@ -36,6 +41,31 @@ body {
     padding: 50px 0;
 }
 
+#ui-domains-list-no-data-small {
+    padding: 10px 0;
+}
+
 #ui-domains-list-template .ui-domain-total {
     font-size: 150%;
 }
+
+#ui-domains-list-detailed-spacer {
+    background-color: #ddd;
+}
+
+#ui-name-uuid {
```

Comment:
Bootstrap already provides classes for text alignment: http://getbootstrap.com/css/#type-alignment.